.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: php
+
+php:
+  - 7.0
+
+env:
+  global:
+    - SIMPLETEST_DB=sqlite://tmp/site.sqlite
+    - SIMPLETEST_BASE_URL="http://127.0.0.1:8080"
+    - DRUSH_PATH="./vendor/bin/drush"
+  matrix:
+    #- RELEASE=stable COMPOSER_CHANNEL=stable
+    - RELEASE=dev COMPOSER_CHANNEL=stable
+    #- RELEASE=stable COMPOSER_CHANNEL=snapshot
+
+before_install:
+  - echo 'sendmail_path = /bin/true' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+  - phpenv config-rm xdebug.ini
+  - composer --verbose self-update --$COMPOSER_CHANNEL
+  - composer --version
+
+install:
+  - composer --verbose install
+
+script:
+  - cd $TRAVIS_BUILD_DIR
+  - $DRUSH_PATH site-install --verbose --yes --db-url=sqlite://tmp/site.sqlite
+  - $DRUSH_PATH runserver $SIMPLETEST_BASE_URL &
```

Comment:
I think making a simple request to the base url is good idea because this could actually fail.  The install via drush doesn't mean you can actually hit the site.